### PR TITLE
An 344 terra console final updates

### PR DESCRIPTION
**Pull request by austinFlipside**
renaming previous PR

### CODE REVIEW

**Review by juls858 — CHANGES_REQUESTED** [review]
there are bad tests:
```
Found 283 models, 2285 tests, 0 snapshots, 1 analysis, 583 macros, 0 operations, 1 seed file, 97 sources, 0 exposures
The selection criterion 'tests/consoles/terra/consoles_terra__aut_peg_variance-asset_no_gap.sql' does not match any nodes
The selection criterion 'tests/consoles/terra/consoles_terra__eut_peg_variance-asset_no_gap.sql' does not match any nodes
The selection criterion 'tests/consoles/terra/consoles_terra__jpt_peg_variance-asset_no_gap.sql' does not match any nodes
The selection criterion 'tests/consoles/terra/consoles_terra__krt_peg_variance-asset_no_gap.sql' does not match any nodes
The selection criterion 'tests/consoles/terra/consoles_terra__sgt_peg_variance-asset_no_gap.sql' does not match any nodes
```

File: models/consoles/terra/consoles_terra__luna_staked_supplyUSD.sql [hunk 1]
```diff
@@ -9,5 +9,6 @@ SUM(balance_usd) AS staked_supply_usd
 FROM 
 {{ ref('terra__daily_balances') }}
 WHERE currency = 'LUNA' AND balance_type = 'staked'
+and address <> 'terra1fl48vsnmsdzcv85q5d2q4z5ajdha8yu3nln0mh'
```
Comment (juls858) [inline models/consoles/terra/consoles_terra__luna_staked_supplyUSD.sql:12]: what is this address for?

**Review by juls858 — CHANGES_REQUESTED** [review]
duplicate file:
consoles_terra__luna_staked_supplyUSD

File: models/consoles/terra/consoles_terra__luna_staked_supplyUSD.sql (hunk 1 above)
Comment (austinFlipside) [reply models/consoles/terra/consoles_terra__luna_staked_supplyUSD.sql:12]: That address is the LUNA foundation address that adds up and stores staked luna

**Review by juls858 — APPROVED** [review]

**Pull request merged**